### PR TITLE
chore: don't run stale bot on forks

### PR DESCRIPTION
**Pull request by lauraseidler**
Forks generally don't have open issues - this PR turns off the stale bot for them, to avoid useless action runs + the notifications arising from them.

Docs: https://docs.github.com/en/actions/learn-github-actions/contexts#github-context

### CODE REVIEW

File: infra/terraform/test-org/github/resources/stale.yml [hunk 1]
```diff
@@ -22,6 +22,7 @@ on:
 
 jobs:
   stale:
+    if: github.repository_owner == 'GoogleCloudPlatform' || github.repository_owner == 'terraform-google-modules'
```
Comment (apeabody) [inline infra/terraform/test-org/github/resources/stale.yml:25]: For the LINT CI

```suggestion
    if: github.repository_owner == 'GoogleCloudPlatform' || github.repository_owner == 'terraform-google-modules'
```

**Review by apeabody — CHANGES_REQUESTED** [review]
Thanks for the contribution @lauraseidler!  Linter found an extra space.

**Comment by apeabody** [comment]
applied

**Comment by lauraseidler** [comment]
Thank you!

**Pull request merged**